### PR TITLE
Create shareuniversity.yml

### PR DESCRIPTION
**Pull request by undershot1**
New tracker ShareUniversity.org. Based on Channelx tracker definition.

### CODE REVIEW

File: src/Jackett.Common/Definitions/shareuniversity.yml [hunk 1]
```diff
@@ -0,0 +1,143 @@
+---
+  site: ShareUniversity
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/shareuniversity.yml]: the site value needs to be the same case sensitive name as the yml filename

File: src/Jackett.Common/Definitions/shareuniversity.yml [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+      - {id: 12, cat: TV/Anime}
+      - {id: 6, cat: Other}
+      - {id: 13, cat: Audio/Audiobook}
+      - {id: 4, cat: Books/Ebook}
+      - {id: 16, cat: E-learning}
+      - {id: 5, cat: PC/Games}
+      - {id: 11, cat: Movies}
+      - {id: 3, cat: Audio}
+      - {id: 10, cat: XXX}
+      - {id: 7, cat: Movies}
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/shareuniversity.yml]: category 7 can not be both tv and movies

File: src/Jackett.Common/Definitions/shareuniversity.yml [hunk 3]
```diff
@@ -0,0 +1,143 @@
+---
+  site: ShareUniversity
+  name: ShareUniversity
+  description: "ShareUniversity is a Private Torrent Tracker for High Quality Nordic Content"
+  language: en-us
+  type: private
+  encoding: UTF-8
+  links:
+    - https://shareuniversity.org
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/shareuniversity.yml]: the link url needs to end with a /

File: src/Jackett.Common/Definitions/shareuniversity.yml [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+    categorymappings:
+      - {id: 1, cat: Movies}
+      - {id: 8, cat: Movies}
+      - {id: 2, cat: TV}
+      - {id: 9, cat: TV}
+      - {id: 12, cat: TV/Anime}
+      - {id: 6, cat: Other}
+      - {id: 13, cat: Audio/Audiobook}
+      - {id: 4, cat: Books/Ebook}
+      - {id: 16, cat: E-learning}
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/shareuniversity.yml]: category 16 is not valid. see https://github.com/nZEDb/nZEDb/blob/dev/docs/newznab_api_specification.txt#L608-L695 for valid values

File: src/Jackett.Common/Definitions/shareuniversity.yml (hunk 2 above)
Comment (undershot1) [reply src/Jackett.Common/Definitions/shareuniversity.yml]: Removed TV, just left it as Movies

**Comment by garfield69** [comment]
Thank you.

**Comment by garfield69** [comment]
available beginning with jackett pre-release 0.11.719

**Pull request merged**